sample/ServiceBusQueueTrigger/index.js

```diff
@@ -1,16 +1,17 @@
-﻿module.exports = function (message, context) {
+﻿module.exports = function (context, message) {
     context.log("Node.js ServiceBus queue trigger function processed message '" + JSON.stringify(message) + "'");
 
-    var output = null;
+    var result = null;
     if (message.count < 1)
     {
         // write a message back to the queue that this function is triggered on
         // ensuring that we only loop on this once
         message.count += 1;
-        output = {
+        result = {
             message: JSON.stringify(message)
         };
     }
+    context.bindings.output = result;
```

Comment:
Is it legal to not set the output at all, instead of setting it to `null` in some cases? If so, you may not need the temporary `result` var, which will look a bit cleaner.